### PR TITLE
[MNO-287] Developer Section in My Account

### PR DESCRIPTION
**Pull request by hedudelgado**
@x4d3  @ouranos

### CODE REVIEW

**Comment by hedudelgado** [comment]
https://github.com/maestrano/mno-enterprise-angular/pull/93

**Comment by hedudelgado** [comment]
https://github.com/maestrano/maestrano-hub/pull/244

**Comment by alexnoox** [comment]
@BrunoChauvet Please review

Frontend: https://github.com/maestrano/mno-enterprise-angular/pull/93

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/current_users_controller.rb [hunk 1]
```diff
@@ -34,6 +33,17 @@ def update
     end
   end
 
+  # PUT /mnoe/jpi/v1/current_user/register_developer
+  def register_developer
```
Comment (BrunoChauvet) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/current_users_controller.rb]: Please write a spec for this method

**Comment by hedudelgado** [comment]
@BrunoChauvet

**Pull request merged**